python/GafferUI/RampPlugValueWidget.py

```diff
@@ -78,35 +78,43 @@ def __init__( self, plug, **kw ) :
 			self.__lastPositionChangedReason = None
 			self.__positionsMergeGroupId = 0
 
+			labelWidth = 50
+
 			with GafferUI.ListContainer(
 				orientation = GafferUI.ListContainer.Orientation.Horizontal if isinstance( plug.pointYPlug( 0 ), Gaffer.FloatPlug ) else GafferUI.ListContainer.Orientation.Vertical,
 				spacing = 4
 			) :
 
-				self.__positionLabel = GafferUI.LabelPlugValueWidget(
-					plug.pointXPlug( 0 ),
-					parenting = { "verticalAlignment" : GafferUI.VerticalAlignment.Top }
-				)
-				self.__positionField = GafferUI.NumericPlugValueWidget(
-					plug.pointXPlug( 0 ),
-					parenting = { "verticalAlignment" : GafferUI.VerticalAlignment.Top }
-				)
-
-				self.__valueLabel = GafferUI.LabelPlugValueWidget(
-					plug.pointYPlug( 0 ),
-					parenting = { "verticalAlignment" : GafferUI.VerticalAlignment.Top }
-				)
-				if isinstance( plug.pointYPlug( 0 ), Gaffer.FloatPlug ):
-					self.__valueField = GafferUI.NumericPlugValueWidget(
-						plug.pointYPlug( 0 ),
+				with GafferUI.ListContainer( orientation = GafferUI.ListContainer.Orientation.Horizontal, spacing = 4 ) :
```

Comment:
I am finding the new layout for ColorSplines a little bit less satisfactory than the old one. I'm wondering if it would make sense to just have a way of turning off the green dots for certain widgets or plugs instead? The idea of a non-default value doesn't make much sense for spline positions/values, at least once you've started adding your own points.

Paging @murraystevenson for a second opinion.

Reply:
Just noting that we talked about this in our weekly meeting and decided to hide the icon for spline position and value plugs. Done in https://github.com/GafferHQ/gaffer/pull/5737/commits/d3aaa2f4146b4b6c848bd0ca02c368db6e1d62e4